From 66db99992e8a432db98203612aa1bf208e941902 Mon Sep 17 00:00:00 2001
From: Lin Huang <hl@rock-chips.com>
Date: Tue, 26 Apr 2016 17:26:32 +0800
Subject: [PATCH] ARM64: dts: rockchip: kevin: add configure for emmc phy

assign freq-sel, dr-sel, opdelay value to meet the hardware
requirement of kevin.

Change-Id: Ibb410c607e69c966a9f2949846ef95ec34e15e79
Signed-off-by: Lin Huang <hl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi
index ca10e259b4e4..2f99031aa237 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-gru.dtsi
@@ -662,6 +662,13 @@ ap_i2c_audio: &i2c8 {
 	status = "okay";
 };
 
+&emmc_phy {
+	freq-sel = <200000000>;
+	dr-sel = <50>;
+	opdelay = <4>;
+	status = "okay";
+};
+
 &sdhci {
 	status = "okay";
 
-- 
2.35.3

